Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2004-11-22
Initial Package Version: 2.6.9.1
Origin: Atsushi Nemoto
Upstream Status: Unknown
Description: Fixes Compile issues with Programs
 
diff -Naur linux-libc-headers-2.6.9.1.orig/include/asm-mips/socket.h linux-libc-headers-2.6.9.1/include/asm-mips/socket.h
--- linux-libc-headers-2.6.9.1.orig/include/asm-mips/socket.h	2004-10-22 21:54:03.000000000 +0000
+++ linux-libc-headers-2.6.9.1/include/asm-mips/socket.h	2004-11-23 04:45:18.911470504 +0000
@@ -68,6 +68,8 @@
 
 #define SO_PEERSEC		30
 
+#ifdef __KERNEL__
+
 /** sock_type - Socket types
  *
  * Please notice that for binary compat reasons MIPS has to
@@ -95,4 +97,6 @@
 
 #define ARCH_HAS_SOCKET_TYPES
 
+#endif /* __KERNEL__ */
+
 #endif /* _ASM_SOCKET_H */
